### PR TITLE
0x Launch kit landing page

### PR DESCRIPTION
**Pull request by fabioberger**
## Description

Desktop: 

localhost_3572_launch-kit ipad pro 1

Mobile:

localhost_3572_launch-kit iphone 6_7_8 3

## Checklist:

*   [ ] Prefix PR title with `[WIP]` if necessary.
*   [ ] Prefix PR title with bracketed package name(s) corresponding to the changed package(s). For example: `[sol-cov] Fixed bug`.
*   [ ] Add tests to cover changes as needed.
*   [ ] Update documentation as needed.
*   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/website/ts/pages/launch_kit/launch_kit.tsx [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+            {
+                icon: '/images/launch_kit/local_market.svg',
+                description: this.props.translate.get(Key.LocalMarket, Deco.Cap),
+            },
+        ];
+        return (
+            <div className="clearfix pb4" style={{ backgroundColor: darkerBackgroundColor }}>
+                <Container
+                    className="mx-auto"
+                    textAlign="center"
```
Comment (fragosti) [inline packages/website/ts/pages/launch_kit/launch_kit.tsx]: This should be in `Text` ?

File: packages/website/ts/pages/launch_kit/launch_kit.tsx [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    icon: string;
+    description: string;
+}
+
+export class LaunchKit extends React.Component<LaunchKitProps, LaunchKitState> {
+    private readonly _throttledScreenWidthUpdate: () => void;
+    constructor(props: LaunchKitProps) {
+        super(props);
+        this.state = {
+            screenWidth: utils.getScreenWidth(),
```
Comment (fragosti) [inline packages/website/ts/pages/launch_kit/launch_kit.tsx]: one day we should put this `screenWidth` logic into a render prop component https://reactjs.org/docs/render-props.html#use-render-props-for-cross-cutting-concerns

**Comment by tomhschmidt** [comment]
image
Vertical alignment looks a bit off here

**Comment by chriskalani** [comment]
Can you change the title halfway down the page to *not* be title cased? We don't do that anywhere else. Should read: `Perfect for developers who need a simple drop-in marketplace`

**Pull request merged**